### ISSUE TITLE
FileNotFoundError when exec tokenizer_build.py

### ISSUE THREAD

**Issue by kleincup** [mandubian/codenets#1]
Hi, @mandubian 
First, thanks for the great work that you have done to reproduce the code in CodeSearchNet with PyTorch.
But when I follow you and test the code in this repo，I had a problem as follows.
When I exec this cmd "python codenets/codesearchnet/tokenizer_build.py --config ./conf/default.conf"
Error happened,
![image](https://user-images.githubusercontent.com/7763863/85940293-1d657f00-b94e-11ea-97b0-7cd6f3445213.png)
It seems that, When I am building the tokenizer, the code is trying to load the tokenizers from the hocon file.
So why load? Could you give me more suggestions?

**Comment by mandubian** [comment]
Thanks! I haven't touched this code for some time and it's a bit draft ;)
I'll check that, I might have solved it on my latest local branch.

**Comment by kleincup** [comment]
Thank you for your response and looking forward that you could check and update your code soon!

**Comment by mandubian** [comment]
hello, I haven't forgotten you but I'm very busy those 2 weeks... I hope you're not in a hurry ;)

**Comment by mandubian** [comment]
Hey @kleincup I've finally found 1h to see your issue... sorry for the veryyyyyyy long delay, I was busy on other topics.
I've pushed a correction here https://github.com/mandubian/codenets/commit/63be72b706d57dbfb2ecec94adc203fc7bdfa3cf 
Actually I've also pushed lots of new code I had forgotten to push.
The issue was that `query_1_code_1` is my first model and I haven't worked on it for long time.
`query_code_siamese` is more up to date.
